pokemon-app/src/components/SinglePokemon.js

```diff
@@ -80,13 +82,32 @@ const SinglePokemon = ({
                 className={`moves-box ${hasEnemy ? "has-enemy" : ""} ${
                   statsOnTop ? "" : "on-top"
                 }`}
+                ref={scrollRef}
+                onScroll={onScroll}
               >
                 <ol id="moves-list">
                   {!pokemon?.moves
                     ? null
                     : pokemon?.moves.map((moveData, moveIndex) => (
                         <li key={moveIndex}>{moveData.move.name}</li>
                       ))}
+
+                  {!pokemon?.moves ? null : !(scrollTop > 0) ? null : (
+                    <span
+                      className={`moves-list--up-arrow ${
+                        hasEnemy ? "has-enemy" : ""
+                      }`}
+                    ></span>
+                  )}
+
+                  {!pokemon?.moves ? null : !(pokemon?.moves.length > 7) ||
```

Comment:
ahhh so you used the moves length to see if it needs arrows in the first place. Very good

Reply:
So clean.